ts/create-live-compositor/templates/node-express-redux/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "node-express-redux",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Live Compositor app",
+  "main": "./dist/index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "ts-node ./src/index.ts",
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@live-compositor/node": "0.1.0-rc.0",
+    "@reduxjs/toolkit": "^2.2.7",
+    "express": "^4.21.0",
+    "live-compositor": "0.1.0-rc.0",
+    "react": "^18.3.1",
+    "react-redux": "^9.1.2"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.21",
+    "@types/node": "^20.14.10",
+    "@types/react": "^18.2.25",
+    "typescript": "^5.5.3"
+  }
+}
```

Comment:
I'm not sure if it's on purpose or not but there is no publicly available `0.1.0-rc.0` version. The latest versions released on npm are `0.1.0-rc.1` or `0.0.1`